src/bootstrap/server-config.js

```diff
@@ -17,15 +17,21 @@ const configSchema = joi.object({
   sessionSecret: joi.string(),
   sessionDurationInMinutes: joi.number().min(1),
   smtpOptions: joi.any(),
-  emailSenderAddress: joi.string(),
+  emailSenderAddress: joi.string().required(),
   publicFolders: joi.array().items(joi.string()),
   resources: joi.array().items(joi.string()),
   initialUser: joi.object({
     username: joi.string().required(),
     password: joi.string().required(),
     email: joi.string().required()
   }).allow(null),
-  exposeErrorDetails: joi.boolean().required()
+  exposeErrorDetails: joi.boolean().required(),
+  importApiKey: joi.string(),
+  importSources: joi.array().items(joi.object({
+    name: joi.string().required(),
+    baseUrl: joi.string().required(),
+    apiKey: joi.string().required()
+  }))
```

Comment:
I'm now a bit confused, since we said it could be optional, shouldn't we allow here .allow(null).

Reply:
Well to me `allow(null)` would make sense when it's mandatory, so that the user has to willingly provide something, even if that something is `null`.
The import properties are optional as they are specified now.